com.unity.multiplayer.mlapi/Runtime/Core/NetworkedBehaviour.cs

```diff
@@ -932,7 +967,7 @@ internal static void SetNetworkedVarData(List<INetworkedVar> networkedVarList, S
 
                     long readStartPos = stream.Position;
 
-                    networkedVarList[j].ReadField(stream);
+                    networkedVarList[j].ReadField(stream, 0, 0);
```

Comment:
I was following along till I got to `0, 0`.  I grasp what `k_NoTick` means, but not `0`.  Can you explain in a comment in the code?

Reply:
good catch. Left-over from before I had `k_noTick`. will fix.